### PR TITLE
Add eslint-plugin-compat

### PR DESCRIPTION
**Pull request by XhmikosR**
Note that we can move the browserlists config to a separate file; I went with package.json so that we don't have one more file.

### CODE REVIEW

File: build/postcss.config.js [hunk 1]
```diff
@@ -5,27 +5,6 @@ module.exports = (ctx) => ({
...
-        'Explorer >= 10',
-        // Out of leniency, we prefix these 1 version further back than the official policy.
-        'iOS >= 9',
-        'Safari >= 9',
-        // The following remain NOT officially supported, but we're lenient and include their prefixes to avoid severely breaking in them.
-        'Android >= 4.4',
-        'Opera >= 30'
-      ]
-    }
+    autoprefixer: {}
```
Comment (Johann-S) [inline build/postcss.config.js]: why did you let that line ? their are nothing in `autoprefixer`

Comment (XhmikosR) [reply build/postcss.config.js]: How will autoprefixer be included otherwise?

**Comment by XhmikosR** [comment]
I ignored the `serviceWorker` errors in pwa.js, but I wonder, should it even complain since we have that block wrapped in a `if ('serviceWorker' in navigator) {` check?

**Comment by XhmikosR** [comment]
@Johann-S: what do we do with this?

**Comment by Johann-S** [comment]
IMO we should merge this PR because currently folks PR aren't tested on Saucelabs so it will help folks to know if their code is compatible with our supported browsers

**Review by Johann-S — APPROVED** [review]

**Pull request merged**